Enhance Modal component to support alertdialog role (Issue/3337) (#3618)
porsche-design-system/porsche-design-system: packages/components/CHANGELOG.md

#### Added

- `Modal`, `Flyout`:

>   - `aria-label` is generated for of slotted header contents if `aria` prop is not provided

Reviewer: ```suggestion
  - `aria-label` is generated from slotted header contents if `aria` prop is not provided
```
